.travis.yml

```diff
@@ -9,33 +9,32 @@ branches:
 before_install:
 - pip install --upgrade pip setuptools wheel
 stages:
-- quick_test
 - test
 - name: deploy
   if: type = push AND branch = master
 matrix:
   include:
-  - stage: quick_test
+  - stage: test
     os: linux
     env: NAME=misc
     python: '3.6'
     install: skip
     script: check/misc
-  - stage: quick_test
+  - stage: test
     os: linux
     env: NAME=pytest-changed-files
     python: '3.6'
     install:
     - dev_tools/conf/pip-install-minimal-for-pytest-changed-files.sh
     script: check/pytest-changed-files
-  - stage: quick_test
+  - stage: test
     os: linux
     env: NAME=format
     python: '3.6'
     install:
       - cat dev_tools/conf/pip-list-dev-tools.txt | grep yapf | xargs pip install
     script: check/format-incremental
-  - stage: quick_test
+  - stage: test
```

Comment:
i modified the travis.yml because the pylint and mypy tests were failing and therefore the builds were not tested. so i created a travis-ci instance to include the build tests anyway.
i did not kniow that this commit was also included in the pull request.